build-and-test.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e 	# Exit immediately upon failure
+set -o pipefail  # Carry failures over pipes
+
+repo_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+docker_repo="microsoft/dotnet-nightly"
+
+pushd "${repo_root}" > /dev/null
+
+find . -path './.git' -prune -o -path './test' -prune -o -path './update-dependencies' -prune -o -name 'Dockerfile' -printf '%p\t%d\n' | egrep -v 'nanoserver|windowsservercore' | sort -n -k2 | while read line
```

Comment:
Ah yes - what do you think about moving the artifacts folder used by test to the root and then excluding just that single folder?

---
In reply to: [76320309](https://github.com/dotnet/dotnet-docker-nightly/pull/35#discussion_r76320309) [](ancestors = 76320309)